### PR TITLE
Djangowebpack

### PR DESCRIPTION
**Pull request by alee046**

### CODE REVIEW

File: flavoUr/settings.py [hunk 1]
```diff
@@ -55,10 +55,12 @@
 
 ROOT_URLCONF = 'flavoUr.urls'
 
+PROJECT_PATH = os.path.realpath(os.path.dirname(__file__))
+
```
Comment (tarastrauss) [inline flavoUr/settings.py]: you can still get rid of this project path :D

File: flavoUr/templates/flavoUr/landing.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  <link rel="stylesheet" href="https://code.getmdl.io/1.1.3/material.indigo-pink.min.css">
+  <script defer src="https://code.getmdl.io/1.1.3/material.min.js"></script>
+</head>
+<body>
+<h1>Welcome to Flavour</h1>
+<h2>Home-made cooking is just <i>next-door</i>.</h2>
+
+<div id="container">
+  {% render_bundle 'main' 'js' %}
+
```
Comment (tarastrauss) [inline flavoUr/templates/flavoUr/landing.html]: extra line!

**Comment by tarastrauss** [comment]
i can barely load this pull request on github lol. lets gitignore the node modules and the flavoUr_env folder? I think that will help and those are setup things that we don't need to include in the repo anyway!

**Comment by alee046** [comment]
ok for sure, yeah i was thinking pushing 3000+ files probably wasn't the right thing LOL

**Pull request merged**